elements/tcpudp/fastudpflows.cc

```diff
@@ -278,6 +283,22 @@ FastUDPFlows_active_write_handler
   return 0;
 }
 
+int
+FastUDPFlows::length_write_handler
+(const String &s, Element *e, void *, ErrorHandler *errh)
+{
+  FastUDPFlows *c = (FastUDPFlows *)e;
+  int len;
+  if (!IntArg().parse(s, len))
```

Comment:
I don't like this element in general; its configury should check that len>=0 and doesn't. But at least the write hadnler should.